PYME/LMVis/renderers.py

```diff
@@ -36,6 +36,7 @@
     'Sample.SlideRef',
     'Sample.Creator',
     'Sample.Notes',
+    'Sample.Well',
```

Comment:
we should probably change this to propagate `Sample.*` (or similar) at some point